diesel/src/pg/expression/expression_methods.rs

```diff
@@ -683,3 +683,592 @@ where
     T::SqlType: RangeOrNullableRange,
 {
 }
+
+/// PostgreSQL specific methods present between CIDR/INET expressions
+pub trait PgNetExpressionMethods: Expression + Sized {
+    /**
+     * Creates a PostgreSQL `>>` expression.
+     *
+     * This operator returns wether a subnet strictly contains another subnet or address.
+     *
+     * # Example
+     *
+     * ```rust
+     * # #[macro_use] extern crate diesel;
+     * # extern crate ipnetwork;
```

Comment:
We don't need any of this anymore here. Master already targets rust-2018 where this just works.

Reply:
Ah, nice! I forgot to remove those when porting from the 1.x PR. Will remove.

Reply:
Removed in bc1bd4a61